config/ignition_fortress_ubuntu_jammy.yaml

```
@@ -0,0 +1,17 @@
+name: ignition_fortress_ubuntu_jammy
+method: http://packages.osrfoundation.org/gazebo/ubuntu-stable
+suites: [jammy]
+component: main
+architectures: [amd64, i386, armhf, arm64, source]
+filter_formula: "\
+(Package (= ignition-cmake2) |\
+ Package (= libignition-cmake2-dev)
+), $Version (% 2.10.0-*) |\
+(Package (= ignition-math6) |\
+ Package (= libignition-math6) |\
+ Package (= libignition-math6-dbg) |\
+ Package (= libignition-math6-dev) |\
+ Package (= libignition-math6-eigen3-dev) |\
+ Package (= python3-ignition-math6)
```

```suggestion
 Package (= python3-ignition-math6) |\
 Package (= ruby-ignition-math6)
```
We now have a nice ruby bindings in ign-math6